src/views/Users/ViewUser/index.jsx

```diff
@@ -445,22 +445,26 @@ function ViewUser({ isNewUser, ...props }) {
             <br />
             <br />
             <br />
-            <Typography variant="h5">Roles</Typography>
-            {roles.map(renderRole)}
-            {additionalRoles.map(renderRole)}
-            <Grid item xs className={classes.addGrid}>
-              <Grid item xs={11}>
-                <Button
-                  onClick={handleRoleAdd}
-                  className={classes.fullWidth}
-                  variant="outlined">
-                  <PlusIcon />
-                </Button>
-              </Grid>
-            </Grid>
-            <br />
-            <br />
-            <br />
+            {!isNewUser && (
+              <Fragment>
+                <Typography variant="h5">Roles</Typography>
+                {roles.map(renderRole)}
+                {additionalRoles.map(renderRole)}
+                <Grid item xs className={classes.addGrid}>
+                  <Grid item xs={11}>
+                    <Button
+                      onClick={handleRoleAdd}
+                      className={classes.fullWidth}
+                      variant="outlined">
+                      <PlusIcon />
+                    </Button>
+                  </Grid>
+                </Grid>
+                <br />
+                <br />
+                <br />
+              </Fragment>
```

Comment:
When the browser is resized in such a way that a user needs to scroll to reach the end of the page, the plus button has no bottom margin:

![Screen Shot 2019-07-31 at 8 39 45 AM](https://user-images.githubusercontent.com/3766511/62212457-c686ac80-b36e-11e9-9a1b-b09b7c07ee6c.png)

This should fix it:

```diff
diff --git a/src/views/Users/ViewUser/index.jsx b/src/views/Users/ViewUser/index.jsx
index ea2521d..5606082 100644
--- a/src/views/Users/ViewUser/index.jsx
+++ b/src/views/Users/ViewUser/index.jsx
@@ -41,6 +41,7 @@ const useStyles = makeStyles(theme => ({
   },
   addGrid: {
     marginTop: theme.spacing(5),
+    marginBottom: theme.spacing(10),
   },
   gridWithIcon: {
     marginTop: theme.spacing(3),
@@ -460,9 +461,6 @@ function ViewUser({ isNewUser, ...props }) {
                     </Button>
                   </Grid>
                 </Grid>
-                <br />
-                <br />
-                <br />
               </Fragment>
             )}
             <Typography variant="h5">Permissions</Typography>
```